assets/stylesheets/_components.scss

```diff
@@ -319,6 +319,7 @@
 @import 'my-sites/all-sites/style';
 @import 'my-sites/all-sites-icon/style';
 @import 'my-sites/comments/style';
+@import 'my-sites/comments/comment/style';
```

Comment:
Is this a good time start splitting out our styles for this section? More info: p4TIVU-8cI-p2.
I can also do this in follow up PR.

Reply:
Great reminder!
We can do it in a follow up, definitely.

Though, we'll have two options:

- Current `CommentDetail` always loaded; future `Comment` loaded on-demand.

- Everything loaded on-demand, with a scss index file including both the list styles, the old detail styles and the new comment styles.

I guess we can discuss this elsewhere though. 🙂